test/test_hs015.py

```diff
@@ -176,7 +176,8 @@ def check_hist_file(self, optimizer, tol):
         val = hist.getValues()
 
         # this check is only used for optimizers that guarantee '0' and 'last' contain funcs
-        if optimizer in ["SNOPT", "SLSQP", "PSQP"]:
+        if optimizer in ["SNOPT", "PSQP"]:
+            print(hist.read("last"))
```

Comment:
Debugging printout?

Reply:
Oh yes good catch, will fix

Reply:
So, we just skip SLSQP check there? What's wrong with the restart?

Reply:
My understanding is that SLSQP does not guarantee that the last call counter contains `funcs`, so I have taken it out of the list of tests here.